Library/Formula/makemake.rb

```diff
@@ -0,0 +1,27 @@
+# makemake formula
+
+class Makemake < Formula
+
+  homepage "https://github.com/Julow/makemake"
+
+  url "https://github.com/Julow/makemake.git", :revision => "a11bba0903aad98765b7fdac0a67ed09903ba669"
```

Comment:
If there's no stable release this is going to need to go in `Homebrew/head-only`.